### PR TITLE
Geofency plugin

### PR DESCRIPTION
**Pull request by Ageman**
Start of a Geofency Plugin.
Geofency is a iPhone App (Maybe Android too, I havn't checkt to be honest) that can send POST requests when iBeacons are in reach or you enter the proximity of a specific location.
I use that to determine if I am at home.
This plugin lets you communicate directly to smarthomeNG to set items like "myPresence" to true if you like.

### CODE REVIEW

File: plugins/geofency/__init__.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        self.sh.trigger ( self.logic, by='geofency', value=data )
+        self._set_headers ( )
+
+def handleRequestsUsing(smarthome, logic):
+    return lambda *args: GeofencyServer(smarthome, logic, *args)
+
+
+class Geofency (SmartPlugin):
+    ALLOW_MULTIINSTANCE = False
+    PLUGIN_VERSION = '0.0.1'
```
Comment (psilo909) [inline plugins/geofency/__init__.py]: please adjust version according to https://github.com/smarthomeNG/smarthome/wiki/Versionsnummern (1.3.0.0.1)

**Comment by psilo909** [comment]
Does it already work? Wont't put a "work in progress" plugin into the repository

File: plugins/geofency/__init__.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    def __init__(self, smarthome, port='2727', logic=''):
```
Comment (ohinckel) [inline plugins/geofency/__init__.py]: Default port should be 2828 as mentioned in the readme - 2727 is already used by the network plugin.

**Comment by Ageman** [comment]
It works, but you are right. I'll get back once it is stabile. I guess some more error handling and stabilization would do good.

**Pull request closed without merging**